includes/Admin/Admin_Page.php

```diff
@@ -94,7 +94,11 @@ public function enqueue_scripts() {
 			'plugin-check-admin',
 			'const PLUGIN_CHECK = ' . json_encode(
 				array(
-					'nonce' => $this->admin_ajax->get_nonce(),
+					'nonce'                               => $this->admin_ajax->get_nonce(),
+					'action_get_checks_to_run'            => Admin_AJAX::ACTION_GET_CHECKS_TO_RUN,
+					'action_set_up_runtime_environment'   => Admin_AJAX::ACTION_SET_UP_ENVIRONMENT,
+					'action_run_checks'                   => Admin_AJAX::ACTION_RUN_CHECKS,
+					'action_clean_up_runtime_environment' => Admin_AJAX::ACTION_CLEAN_UP_ENVIRONMENT,
```

Comment:
Better to use camel case for the JS properties as that is in line with WP JS coding standards (e.g. `PLUGIN_CHECK.actionGetChecksToRun`).

Reply:
Thanks @felixarntz I've updated these to camel case